utilities/infra.py

```
@@ -211,7 +211,8 @@ def wait_for_inference_deployment_replicas(
     expected_num_deployments: int = 1,
     labels: str = "",
     deployed: bool = True,
-    timeout: int = Timeout.TIMEOUT_5MIN,
+    #timeout: int = Timeout.TIMEOUT_5MIN,
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)